src/topology/locally_constant/algebra.lean

```diff
@@ -151,4 +152,7 @@ function.injective.distrib_mul_action coe_fn_add_monoid_hom coe_injective (λ _
 instance [semiring R] [add_comm_monoid Y] [module R Y] : module R (locally_constant X Y) :=
 function.injective.module R coe_fn_add_monoid_hom coe_injective (λ _ _, rfl)
 
+instance [comm_semiring R] [semiring Y] [algebra R Y] : algebra R (locally_constant X Y) :=
+algebra.of_module' (by { intros, ext, simp, }) (by { intros, ext, simp, })
```

Comment:
I don't think this is a good idea, as it doesn't pick a definitionally convenient algebra map.

Reply:
OK done.